Formula/rgbds.rb

```diff
@@ -14,8 +14,11 @@ class Rgbds < Formula
     sha256 "c08b856ecf4cb57390ec99241d0bc87ba623536ab7e3e11d5cc23334230ff1cd" => :mavericks
   end
 
+  depends_on "pkg-config" => :build
```

Comment:
Although it looks like upstream has hardcoded `pkg-config` in their Makefile, so we'll need it anyways. Ignore the removal bit, but the rest of the previous comment still applies.